docker/candlepin-base/cp-test

```diff
@@ -110,6 +110,17 @@ build_rspec() {
     fi
 }
 
+build_spec() {
+    # TODO: Add support for filtering spec tests once it's supported
+
+    # Check that both gradlew exists and the task exists
+    if ! test -f ./gradlew || ! ./gradlew tasks | grep -q "^spec\b"; then
```

Comment:
There's no need to check that gradlew exists any more; any branches that don't have gradle are no longer supported

Reply:
The gradlew check is mostly defensive programming and ensuring that we're in the right directory for the reference (since older builds which do support it may have us in the server directory).

Unless you see it hurting something, I'd feel better leaving it there.

Reply:
You're right. This will run in older branches too, so if the java spec isn't there, we should return but not fail

Reply:
In the latest run, java spec tests aren't running, and the `Warning: Java spec tests not found` is shown.

Reply:
ARRRRRRRGH, my test code to verify what happens when the task doesn't exist got left in. Fixed.